crates/solver/src/solver.rs

```diff
@@ -282,6 +283,7 @@ pub fn create(
     order_prioritization_config: &single_order_solver::Arguments,
     post_processing_pipeline: Arc<dyn PostProcessing>,
     domain: &DomainSeparator,
+    mut s3_instance_uploader_for_quasimodo: Option<S3InstanceUploader>,
```

Comment:
Quasimodo is disabled on Mainnet AFAIK. I think this uploader may not be used at all in production.

Reply:
Wait, doesn't this pretend it is used for Quasimodo so it is just creating subinstances appropriate for Quasimodo? I.e., does it really require that the instance is sent to Quasimodo itself? I thought it would in any case upload, which tbh, it still very useful as we use Quasimodo for debugging and checking certain solutions, if though, as you said, Quasimodo is NOT used as a solver on mainnet anymore.